src/metalsProtocol.ts

```diff
@@ -60,3 +60,9 @@ export namespace MetalsDidFocus {
     "metals/didFocusTextDocument"
   );
 }
+
+export interface MetalsNewScalaFileParams {
+  directory?: string;
+  name: string;
+  kind: string;
```

Comment:
can this be more specific? 

```suggestion
  kind: 'class' | 'object' | 'trait' | 'package-object' | 'worksheet';
```

Reply:
This is marked as resolved but it seems to be unchanged

Reply:
Ah, sorry. I added exactly what you wanted right below this and didn't actually update this. Fixed now.